README-HE.md

```diff
@@ -175,7 +175,6 @@
 
 ## יוזמות לתרומת קוד פתוח
 - [Up For Grabs](https://up-for-grabs.net/) - מכיל פרויקטים עם issues המתאימים למתחילים.
-- [First Timers Only](https://www.firsttimersonly.com/) - רשימה של באגים המתוייגים כ-״first-timers-only".
```

Comment:
This can be kept and added back in.

Reply:
done add66b602f87077fb9a28bfc9c24e09203be26b5